specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/virtualNetworkGateway.json

```diff
@@ -2775,6 +2823,90 @@
         "name": "VirtualNetworkGatewayConnectionType",
         "modelAsString": true
       }
+    },
+    "VpnClientConnectionHealthDetailListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VpnClientConnectionHealthDetail"
+          },
+          "description": "List of vpn client connection health"
+        }
+      },
+      "description": "List of virtual network gateway vpn client connection health"
+    },
+    "VpnClientConnectionHealthDetail": {
```

Comment:
Shouldn't this be plural, "Detail **s** "?

Reply:
no "s".  the above array is an array of VpnClientConnectionHealthDetail